pages/common/prettier.md

```diff
@@ -0,0 +1,24 @@
+# prettier
+
+> An opinionated code formatter for JavaScript, JSON, CSS, YAML, and more.
+> More information: <https://prettier.io/>.
+
+- Format a file and print the result to stdout:
+
+`prettier {{path/to/file}}`
+
+- Check if a specific file has been formatted:
+
+`prettier --check {{path/to/file}}`
+
+- Run with a specific configuration file:
+
+`prettier --config {{path/to/config_file}} {{path/to/file}}`
+
+- Format a file, replacing the original:
+
+`prettier --write {{path/to/file}}`
+
+- Use single quotes instead of double quotes and no trailing commas when formatting a file:
+
+`prettier --single-quote --trailing-comma {{none}} --write`
```

Comment:
Should we use `path/to/file_or_directory` here?

Reply:
Same questiong.

Reply:
Hrm. Why the `none` here?

Reply:
@sbrl the `none` is just for example, to show some command line formatting options.

Reply:
@sbrl because we don't like trailing commas in JavaScript :rofl: